jbdevstudio/com.jboss.jbds.central.discovery.earlyaccess/plugin.xml

```diff
@@ -75,5 +65,30 @@ This content is unavailable. Enable Early Access to see it.</description>
          <overview url="http://www.jboss.org/tools"/>
       </connectorDescriptor>
 
+       <!-- JBoss Fuse Development -->
```

Comment:
but you now risk having these not-worthy for release burned into master. 

Will this be fixed for the CR1 testing at least ?